clio-client/src/test/scala/org/broadinstitute/clio/client/util/MockIoUtil.scala

```diff
@@ -3,11 +3,14 @@ package org.broadinstitute.clio.client.util
 import java.net.URI
 
 import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.clio.client.util.IoUtil.GsUtil
 
 import scala.collection.mutable
 
 class MockIoUtil extends IoUtil with LazyLogging {
 
+  override val gsUtil: GsUtil = new GsUtil(None)
```

Comment:
It looks like all uses of `GsUtil` construct it using `GsUtil(None)`. If so, we should remove the constructor argument.